docs/changes/0.19.0.rst

```diff
@@ -0,0 +1,74 @@
+Changelog for QCoDeS 0.19.0
+===========================
+
+The October 2020 release of QCoDeS.
+
+
+Breaking Changes:
+_________________
+
+There are no breaking changes in this release of QCoDeS.
+
+
+New:
+____
+-  Notebook: Example of how buffered sweep could be done on param with setpoints (#2263)
+
+
+Improved:
+_________
+- Station: Take verbose error message when validating station yaml against schema (#2282)
+- Legacy: ensure that datasets have unique names to avoid potential race condition (#2276) 
+- Add optional shape metadata to dataset (#2145)
+
+Improved drivers:
+_________________
+- AMI430: `has_current_rating` has been removed in the driver to work with the new firmware version (#2291)
+- Small typo fix in alazer drivers (#2296) 
+- Yokogawa GS200: `source_mode` bug fix (#2143)
+
+
+Under the hood:
+_______________
+- Expand document requirments (#2289) MORE_INFO
+- Small improvements in legacy data notebook (#2295)
+- Pylint and it's dependencies removed from test_requirements (#2297)
+- Station example doc: update update_config_schema example to refer to qcodes_contrib_drivers (#2272)
+- Fix some of the warnings triggered by numpy 1.19 (#2239)
+- Fix sequence sorted check to work correctly with sequence types other than list (#2271)
+- Move tests related to qcodes.data loop etc to a legacy folder (#2238)
+
+
+
+Dependencies:
+_____________
+
```

Comment:
I would clarify that this section only related to the requirements.txt files and not setup.py